src/api.py

```diff
@@ -312,6 +368,8 @@ def gather_data(lat, long, arguments):
 
     wind_temp = current_wind_temp(lat, long, arguments["decimal"])
 
+    hourly_dict = get_hourly_forecast(lat, long, arguments["decimal"])
```

Comment:
You call `get_hourly_forecast` here and also in `cli.py`. It should only be called here, in `gather_data`